perfkitbenchmarker/providers/rackspace/flags.py

```diff
@@ -14,37 +14,39 @@
 
 from perfkitbenchmarker import flags
 
-flags.DEFINE_boolean('boot_from_cbs_volume', 'False',
-                     'When flag is included the instance will use a remote disk'
-                     'as its boot disk, if machine_type supports it.')
+flags.DEFINE_string('rack_path',
+                    default='rack',
+                    help='The path for the rack CLI binary.')
 
-flags.DEFINE_string('nova_path',
-                    'nova',
-                    'The path for the rackspace-novaclient tool.')
+flags.DEFINE_string('region', default='IAD',
```

Comment:
Please rename to `--rackspace_region` or something like that, to avoid confusion over the fact that it only applies to Rackspace.